examples/vm_scheduling/reinforcement_learning/ac/components/env_wrapper.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import pdb
+import scipy.signal
+import numpy as np
+from collections import defaultdict
+
+from maro.rl import ExperienceSet
+from examples.vm_scheduling.refine_rl.common import VMEnvWrapper
+
+
+class VMEnvWrapperForAC(VMEnvWrapper):
+    def get_experiences(self):
+        buf = self._replay_buffer
+
+        rewards = np.array(buf["rewards"])
+        cumsum_rewards = self.discount_cumsum(rewards, self._gamma)
+
+        exp_set = ExperienceSet(
+            buf["states"][:-1],
+            buf["actions"][:-1],
+            cumsum_rewards[:-1],
+            buf["states"][1:],
+            buf["info"][1:],
+        )
+        del buf["states"][:-1]
+        del buf["actions"][:-1]
+        del buf["rewards"][:-1]
+        del buf["info"][:-1]
```

Comment:
return `buf["info"][1:]` but `del buf["info"][:-1]`?

Reply:
info store the legal_action; In DQN, exp_set should store the next_legal_action just like the next_states. But legal_action is useless in the AC training process. So I use the same treatment as the DQN.

Reply:
If it is useless, why add it into `buf["info"]` then?